src/test/java/seedu/address/logic/commands/DeleteScoreCommandTest.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_SCORE;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_STUDENT;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_SCORE;
+import static seedu.address.testutil.TypicalStudents.getTypicalMathutoring;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+class DeleteScoreCommandTest {
+
+    private Model model = new ModelManager(getTypicalMathutoring(), new UserPrefs());
+
+    /*@Test
+    public void constructor_nullIndexes_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new DeleteScoreCommand(null, null));
+    }*/
```

Comment:
Can remove this part if it is not used

Reply:
To keep this part or not is depends on whether Qianhui wanna change part of the java code. The test case is valid, but fail cause on line missing in deleteScoreCommand.

Reply:
Ohh I see, I see. Okay!

Reply:
@QQH0828 will you be changing part of the java code to allow for the addition of this test case?

Reply:
Yes, I will add this Java code after confirming with Yu Ting.